packages/worker/src/api/controllers/global/users.ts

```diff
@@ -84,12 +142,67 @@ export const adminUser = async (ctx: any) => {
 
 export const destroy = async (ctx: any) => {
   const id = ctx.params.id
+  const db = tenancy.getGlobalDB()
+  let user: User = await db.get(id)
+  let groups = user.userGroups
+
   await users.destroy(id, ctx.user)
+
+  // Remove asssosicated groups
+  if (groups) {
+    let groupsPromises = []
+    for (const groupId of groups) {
+      let group = await db.get(groupId)
+      let updatedUsersGroup = group.users.filter(
+        (groupUser: any) => groupUser.email !== user.email
```

Comment:
Should probably use `_id` here for additional safety for when we roll out ability to update email addresses. Group users would be another thing we need to update it we didn't use id here